pkg/api/types.go

```diff
@@ -1656,6 +1656,11 @@ func (k *KubernetesConfig) IsAADPodIdentityEnabled() bool {
 	return k.IsAddonEnabled(AADPodIdentityAddonName)
 }
 
+// IsContainerMonitoringAddonEnabled checks if the container monitoring addon is enabled
+func (k *KubernetesConfig) IsContainerMonitoringAddonEnabled() bool {
+	return k.IsAddonEnabled(ContainerMonitoringAddonName)
```

Comment:
can you add a unit test for this? See TestIsClusterAutoscalerEnabled for an example

Reply:
added tests for the container monitoring addon.

Reply:
> can you add a unit test for this? See TestIsClusterAutoscalerEnabled for an example

Done.added tests for the container monitoring addon.